### DISCUSSION TITLE
Backup iPhone

### DISCUSSION THREAD

**Discussion by vitotai** [outdoorbits/little-backup-box#304] [General]
Anybody has done this?
Mine doesn't work. I SSH and try gphoto2 command.

The following command returns 0 files.
```
 gphoto2 --list-files
```

However, manually mounting by fusermount, there are image files under DCIM, and I was able to run rsync to backup the images of my iPhone.

Any idea?

**Comment by outdoorbits** [comment]
Does lbb find your phone at all? Please have a look at the web UI, sysinfo. Is there a camera visible? 
I don't know apple devices, but did you allow USB to use PTP?

**Reply by vitotai** [reply]
YES. The iPhone was found. I and not aware of PTP option for USB.
